### PR TITLE
Bug 1887596: Fix ovs-configuration detecting bond and vlan interfaces

### PR DESCRIPTION
**Pull request by trozet**
When we add the bond port OVS connection it needs to use type bond
instead of ethernet. Similarly VLAN needs type vlan as well as some
extra arguments.

Signed-off-by: Tim Rozet <trozet@redhat.com>
(cherry picked from commit 64b79dfb1247e480478633f3f383653a6f787d3b)

### CODE REVIEW

**Comment by trozet** [comment]
@danwinship PTAL

**Review by cgwalters — APPROVED** [review]
At some point soon I may get motivated to demo how this could use the MCD code in Go...

/approve

**Comment by sinnykumari** [comment]
both OKD tests are known to be failed due to https://github.com/coreos/fedora-coreos-tracker/issues/649
/skip

**Comment by danwinship** [comment]
/hold
all the required tests are passing but the bot is going to keep retrying the optional broken test until this gets `cherrypick-approved`

**Comment by kikisdeliveryservice** [comment]
> /hold
> all the required tests are passing but the bot is going to keep retrying the optional broken test until this gets `cherrypick-approved`

should be good now

**Comment by trozet** [comment]
@kikisdeliveryservice can we override e2e-gcp-op? These changes only affect OVN and that is an openshift-sdn deployment.

**Pull request merged**
